packages/infra/lib/ihe-stack.ts

```diff
@@ -195,12 +217,16 @@ export class IHEStack extends Stack {
       envType: props.config.environmentType,
       envVars: {
         MEDICAL_DOCUMENTS_BUCKET_NAME: props.config.medicalDocumentsBucketName,
+        ...(posthogSecretName ? { POST_HOG_API_KEY: posthogSecretName } : {}),
```

Comment:
This env var name is misleading, it doesn't contain the API key, but the secret name. Maybe `POSTHOG_API_KEY_SECRET_NAME`?